fields-livedata/src/commonMain/kotlin/dev/icerock/moko/fields/livedata/BaseLiveDataFormField.kt

```diff
@@ -55,6 +55,11 @@ abstract class BaseLiveDataFormField<D, E>(
         showValidationError.value = true
         return isValid.value
     }
+
+    override fun clear() {
+        setValue(initialValue)
+        showValidationError.value = false
+    }
```

Comment:
maybe you should also add the reset validation function?

Reply:
```

Reply:
```kotlin
fun resetValidation() {
    showValidationError.value = false
}
```

Reply:
yes, the function of resetting validation without deleting the value can also be useful